Dockerfile

```diff
@@ -3,7 +3,19 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ############# golang-base
-FROM eu.gcr.io/gardener-project/3rd/golang:1.16.2 AS golang-base
+FROM golang:1.16.2 AS golang-base
+
+############# terraform-config
+FROM golang-base AS terraform-config
+
+# overwrite to build provider-specific image
+ARG PROVIDER=all
+
+# Get entire dir to preserve the symblic links to TF_VERSION files
+COPY . .
```

Comment:
Can we avoid copying the entire directory (including `vendor` and so on) and instead only copy the `build` dir / relevant files?

Reply:
Done.